pkg/configuration/configuration.go

```diff
@@ -144,37 +145,24 @@ func ValidateParams(config *Config) error {
 	}
 	// We need to either have crowdsecLapiKey defined or the BouncerCert and Bouncerkey
 	if lapiKey == "" && (certBouncer == "" || certBouncerKey == "") {
-		return fmt.Errorf("CrowdsecLapiKey || (CrowdsecLapiTLSCertificateBouncer && CrowdsecLapiTLSCertificateBouncerKey): cannot be both empty")
+		return fmt.Errorf("CrowdsecLapiKey || (CrowdsecLapiTLSCertificateBouncer && CrowdsecLapiTLSCertificateBouncerKey): cannot be both false")
 	} else if lapiKey != "" && (certBouncer == "" || certBouncerKey == "") {
-		// check LAPIKey
-		lapiKey = strings.TrimSuffix(lapiKey, "\n")
-		err = validateParamsAPIKey(lapiKey)
-		if err != nil {
+		lapiKey = strings.TrimSpace(lapiKey)
```

Comment:
Why this change ?  
This does not solve the same issue.  
We could consider removing spaces + end of line caracters